docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+version: "3.7"
+services:
+  nginx:
+    image: nginx:1.19-alpine
+    ports:
+      - "80:80"
+    volumes:
+      - ./configurations/30-sl-nginx.entrypoint.sh:/docker-entrypoint.d/30-sl-nginx.entrypoint.sh:ro
+  postgres: # Must be identical to the .env POSTGRES_HOST variable
+    image: postgres:12.1
+    environment:
+      POSTGRES_USER: ${POSTGRES_DB:?Required}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:?Required}
+      POSTGRES_DB: ${POSTGRES_DB:?Required}
+      DB_HOST: ${POSTGRES_HOST:?Required}
+  postfix: # temporary
+    image: arugifa/simplelogin-postfix:3.1.0-6
+    ports:
+      - "25:25"
+    environment:
+      POSTFIX_FQDN: ${EMAIL_DOMAIN}
+      ALIASES_DEFAULT_DOMAIN: ${EMAIL_DOMAIN}
+      DB_NAME: ${POSTGRES_DB}
+      DB_HOST: ${POSTGRES_HOST}
+      DB_USER: ${POSTGRES_USER}
+      DB_PASSWORD: ${POSTGRES_PASSWORD}
+      LETSENCRYPT_EMAIL: admin@sl.com
+      EMAIL_HANDLER_HOST: email-handler # Must be identical to the service name
+    volumes:
```

Comment:
postfix doesn't need this `volume` right?

Reply:
oops, wrong indentation, it's been fixed